app/src/main/java/com/eventyay/organizer/core/auth/start/StartViewModel.java

```diff
@@ -37,6 +37,8 @@
     ;
     private final SingleEventLiveData<Boolean> isEmailRegistered = new SingleEventLiveData<>();
     private final MutableLiveData<EmailRequest> emailRequestModel = new MutableLiveData<>();
+    private final MutableLiveData<Boolean> showWarningDialog=new MutableLiveData<>();
+    private static final String warningPreferenceKey = "experimental_warning_shown";
```

Comment:
Capitalize

Reply:
Capitalizing the String will not adhere to the spotlessJava lint . As it is following Camel Case in the Whole Project.Thank You

Reply:
Or Writing **WARNINGPREFERENCEKEY** would be accepted?